src/contracts/interfaces/IEigenPod.sol

```diff
@@ -50,11 +56,8 @@ interface IEigenPod {
     /// @notice The amount of eth, in wei, that is restaked per validator
     function REQUIRED_BALANCE_WEI() external view returns(uint256);
 
-    /// @notice this is a mapping of validator indices to a Validator struct containing pertinent info about the validator
-    function validatorStatus(uint40 validatorIndex) external view returns(VALIDATOR_STATUS);
-
     /// @notice the amount of execution layer ETH in this contract that is staked in EigenLayer (i.e. withdrawn from beaconchain but not EigenLayer), 
-    function restakedExecutionLayerGwei() external view returns(uint64);
+    function withdrawableRestakedExecutionLayerGwei() external view returns(uint64);
```

Comment:
I will read further, but is this even the correct name any more? are these ETH 'restaked' or are they just in the withdrawal 'limbo' kind of state?

Reply:
It functions in pretty much the same way